tests/phpstan.neon

```diff
@@ -0,0 +1,10 @@
+parameters:
+    paths:
+        - %currentWorkingDirectory%/module
+    tmpDir: %currentWorkingDirectory%/.phpstan_cache
+    ignoreErrors:
+        - '#Class VuFind\\Hierarchy\\Driver\\Hierarchy[a-zA-Z0-9]+ not found#'
+        - '#Class VuFind\\Content\\[a-zA-Z0-9]+\\SyndeticsPlus not found#'
+        - '#Class VuFind\\Content\\[a-zA-Z0-9]+\\Amazon[a-zA-Z0-9]* not found#'
+        - '#Function sybase_[a-z_]+ not found#'
```

Comment:
One other question: does PHPStan support ignoring errors through annotations in the code? I wonder if it would be better to annotate the offending fiels rather than globally ignoring specific errors. That way, if problems get resolved or go away (for example, if we get rid of all sybase function calls in the future), we don't have to remember to come back here and change this.

Reply:
Yes, PHPStan supports this. I could rework this and use inline annotations to ignore errors.

Reply:
Good to know! I'll leave it to your discretion, then. If there are errors that we might really want to know about (like use of deprecated functions), annotations still might be better... but for things that are widespread and intentional, keeping them here makes sense!